backend/src/test/java/com/woowacourse/friendogly/footprint/domain/LocationTest.java

```diff
@@ -1,12 +1,44 @@
 package com.woowacourse.friendogly.footprint.domain;
 
 import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatCode;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
+import com.woowacourse.friendogly.exception.FriendoglyException;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
 
 class LocationTest {
 
+    @DisplayName("생성 성공 - 올바른 위도, 경도 범위")
+    @ParameterizedTest
+    @CsvSource(value = {"90.000, 180.000", "-90.000, -180.000", "0.000, 0.000"})
+    void constructor_Success(double latitude, double longitude) {
+        assertThatCode(() -> new Location(latitude, longitude))
+            .doesNotThrowAnyException();
+    }
+
+    @DisplayName("생성 실패 - 올바르지 않은 위도 범위")
+    @ParameterizedTest
+    @ValueSource(doubles = {-90.001, 90.001})
+    void constructor_Fail_IllegalLatitude(double latitude) {
+        assertThatThrownBy(() -> new Location(latitude, 0.0))
+            .isInstanceOf(FriendoglyException.class)
+            .hasMessage("위도 범위가 올바르지 않습니다.");
+    }
+
+    @DisplayName("생성 실패 - 올바르지 않은 위도 범위")
```

Comment:
```suggestion
    @DisplayName("위도 범위를 벗어난 값을 입력하는 경우 예외가 발생한다.")
```
`@DisplayName` 은 컨벤션에 맞춰 작성해주세요!